public/data/khushaal.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Khushaal Choithramani",
+  "bio": "Software Developer || Coder || Open Source Enthusiast",
+  "avatar": "https://avatars.githubusercontent.com/u/73185436?s=400&u=3fe5a63d40986567c44bb98e277a081d21991077&v=4",
```

Comment:
```suggestion
  "avatar": "https://github.com/khushxxl.png",
```

Reply:
#265, image fixed in this pr

Reply:
But why did @khushxxl create another PR for that? 😕

Reply:
I am not sure, maybe close this one, then review other.

@khushxxl you can simply change and alter your work in the same pr, it just adds another commit, with your change.

Reply:
I have closed my another pr and will further fix this pr only . I am sorry i am new to contributing!
will fix it now!